### PR TITLE
feat: Modify the menu icon

### PR DESCRIPTION
**Pull request by cccccccb**
Change the icon of menu button to the Dci icon.

Log: Change icon
Influence: Menu button
Change-Id: Ie31f8e3e6c7141e340e4613e2f15b54b8faa83ea

### CODE REVIEW

File: src/widgets/dstyle.cpp [hunk 1]
```diff
@@ -649,12 +652,14 @@ void drawTitleBarMenuButton(QPainter *pa, const QRectF &rect)
     content_rect.moveCenter(rect.center().toPoint());
     pa->setPen(pen);
 
-    pa->setRenderHint(QPainter::Antialiasing, pa->device()->devicePixelRatioF() > 1.0);
-    pa->drawLine(content_rect.x(), content_rect.y(), content_rect.topRight().x() - 2, content_rect.topRight().y());
-    pa->drawLine(content_rect.bottomLeft(), content_rect.bottomRight());
-
-    qreal y = content_rect.center().y();
-    pa->drawLine(content_rect.x(), y, content_rect.topRight().x(), y);
+    const char menubarIconName[] = "window_menu";
```
Comment (kegechen) [inline src/widgets/dstyle.cpp:655]: QLatin1String 可以吧

Comment (cccccccb) [reply src/widgets/dstyle.cpp]: 已修改

**Review by kegechen — APPROVED** [review]

**Pull request merged**